mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/French.kt

```diff
@@ -108,6 +108,7 @@ object French : LocalizedStrings {
       "Trouve un joueur en utilisant son nom, suis le, invite-le à jouer ou consulte l'historique de ses matchs."
   override val socialSearchClearContentDescription = "Effacer le champ de recherche"
   override val socialCloseVisitedProfile = "Fermer"
+  override val socialUnfollowIcon = "Desuivi"
```

Comment:
Je pense que "Supprimer" correspond mieux à l'action ^^

```suggestion
  override val socialUnfollowIcon = "Supprimer"
```